components/server/src/server_utilities/functions.py

```diff
@@ -7,7 +7,7 @@
 from collections.abc import Callable, Hashable, Iterable, Iterator
 from datetime import datetime, timezone
 from decimal import ROUND_HALF_UP, Decimal
-from typing import Tuple, TypeVar, cast
+from typing import Dict, List, Tuple, TypeVar, cast
```

Comment:
```suggestion
from typing import Tuple, TypeVar, cast
```

(Tuple can be removed as well...)